format_notification.py

```diff
@@ -0,0 +1,72 @@
+#!/bin/env python
+
+# this while loop is for not considering two string fields which you get right after starting a process
+# Test string ->['string "ShanonPark posted in "HEY! YOU! What are you up to right now! (Part 3)" - Linkin Park / LPU Forums"'] 2 
+
+import time
+import sys
+import notify2
+from setup import main
+
+def EditString(data: str) -> str:
+    return data[0].split('"', 1)[1].replace('"', '')
+
+def Notify(summary, data, urgency, timeout=800000):
+    notify2.init("AutoReplyToLP")
+    pop_up = notify2.Notification(summary, data)
+    pop_up.set_urgency(urgency)
+    pop_up.set_timeout(timeout)
+    return pop_up
+
+def countdown_timer(timeMS):
+    # timeMS should be in millisecond
+    minute = timeMS // 60000
+    second = 59
+    pop_up = Notify(f"{minute:02d} : {second:02d}", "Time Left", 1)
+
+    while minute > -1:
+        pop_up.update(f"Time Left -> {minute:02d} : {second:02d}")
+        time.sleep(1)
+        pop_up.show()
+        if second == 0:
+            minute -= 1
+            second = 60
+        second -= 1
+    pop_up.update("Countdown Completed")
+    pop_up.show()
+
+wasteLines = 0
```

Comment:
comment saying why we need to waste lines is nice imo

Reply:
looks like we only need to skip over two lines?

Reply:
yes, we need to skip the first two lines right after we captured the first notification because the first two lines represent service names as a string. 
This skip will be done only after capturing the first notification and for the rest of notification, it won't be call again.